<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="deviceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>COM Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="commandFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Command File</string>
    </property>
   </widget>
   <widget class="QLabel" name="logFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Log File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commandFileBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logFileBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="commentLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>284</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="connectionBtnLayout">
     <item>
      <widget class="QRadioButton" name="connectionBtnSSH">
       <property name="text">
        <string>SSH</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">connectionBtnGr</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connectionBtnTelnet">
       <property name="text">
        <string>Telnet</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">connectionBtnGr</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connectionBtnSerial">
       <property name="text">
        <string>Serial</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">connectionBtnGr</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connectionBtnMacro">
       <property name="text">
        <string>Depend on Macro</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">connectionBtnGr</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="messageText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="macroLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro File</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="commandListTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>721</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="programLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Program File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="comText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandFileText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="logFileText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="commentText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="programText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="macroText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="programTextEnableCheck">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable program file modify</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="macroTextEnableCheck">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable macro file modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="macroFileBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="debugModeCheck">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programFileBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>161</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>228</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="logFileOptionBtnLayout">
     <item>
      <widget class="QCheckBox" name="logAppendCheck">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="commandListLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Command List</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="deviceBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ipBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="usernameBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="passwordBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="commandFileBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="logFileBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="commentBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="programBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="macroBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>deviceText</tabstop>
  <tabstop>programText</tabstop>
  <tabstop>programFileBtn</tabstop>
  <tabstop>programTextEnableCheck</tabstop>
  <tabstop>ipText</tabstop>
  <tabstop>macroText</tabstop>
  <tabstop>macroFileBtn</tabstop>
  <tabstop>macroTextEnableCheck</tabstop>
  <tabstop>comText</tabstop>
  <tabstop>connectionBtnSSH</tabstop>
  <tabstop>connectionBtnTelnet</tabstop>
  <tabstop>connectionBtnSerial</tabstop>
  <tabstop>connectionBtnMacro</tabstop>
  <tabstop>usernameText</tabstop>
  <tabstop>passwordText</tabstop>
  <tabstop>commandFileText</tabstop>
  <tabstop>commandFileBtn</tabstop>
  <tabstop>logFileText</tabstop>
  <tabstop>logFileBtn</tabstop>
  <tabstop>logAppendCheck</tabstop>
  <tabstop>commentText</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>debugModeCheck</tabstop>
  <tabstop>commandListTable</tabstop>
  <tabstop>messageText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="connectionBtnGr"/>
 </buttongroups>
</ui>
